acropolis-persistence/src/main/kotlin/org/ephyra/acropolis/persistence/api/entity/GraphicalAssetEnttiy.kt

```diff
@@ -0,0 +1,53 @@
+package org.ephyra.acropolis.persistence.api.entity
+
+import org.ephyra.acropolis.persistence.api.GraphicalAssetType
+import java.util.Arrays
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.Lob
+
+/**
+ * Entity model to represent a graphical asset (e.g. an icon)
+ */
+@Entity
+data class GraphicalAssetEnttiy @JvmOverloads constructor(
```

Comment:
Just in case you hadn't spotted it yet. Typo in the name here. `enttiy` instead of `entity`

Reply:
Totally deliberate, it's actually old norse for entity :P Good spot thanks! This is why opening PRs early is so useful :-)